README.md

```diff
@@ -569,6 +576,43 @@ const date = Date.now();
 - Add another **registerLocale** with your language as the first parameter and the import from `date-fns` as second parameter.
 - Place your language with its date format on **dateFormat**.
 
+## File Uploading
+
+For file uploading, we used the [Firebase Client](https://firebase.google.com/docs/storage/web/upload-files?authuser=1#upload_files) in conjuntion with **Firebase Storage**. We store the users profile images in a subfolder called `/users`. Besides, we decided to use the firebase storage as a bucket to store files.
+
+### Image Resize
+
+We use [Resize Image](https://github.com/firebase/extensions/tree/master/storage-resize-images) for resizing every image uploaded to the storage.
+Every image uploaded to the storage is resized to our size preference (`200px x 200px`).
+
+**Setting your image resize**
+
+<ol>
+<li>Go to your Firebase Proyect on <a href= "https://console.firebase.google.com/u/1/">Firebase Console</a> </li>
+<li>Select extensions from the sidebar</li>
+<li>Search for Resize Images extension</li>
+<li>Click install on the extension</li>
+<li>On extension configuration set sizes of resized images to 200x200 and the deletion of original file to true</li>
+<li>Click on install extension</li>
+</ol>
+
```

Comment:
If you want to install it from the cmd, you can execute:
```
firebase ext:install storage-resize-images --project=projectId
```
with your own `projectId`